_posts/2017-07-10-make-async-await-work-in-promises.md

```diff
@@ -35,9 +35,16 @@ But still no luck, then I realised I didn't have an async function. The arrow fu
 
 ```js
 function returnSomething(name) {
```

Comment:
Hm, why not this then:

```js
async function returnSomething(name) {
  return returnSomethingElse();
}
```

Reply:
Yes . This should work but the topic is about using async and await within Promises. 
Here's a link explaining why you shouldn't use a async in a promise which may lead to difficulties in debugging errors.
[Ref](https://eslint.org/docs/rules/no-async-promise-executor#:~:text=If%20an%20async%20executor%20function,debug%20and%20handle%20some%20errors.)